pyaedt/generic/plot.py

```diff
@@ -378,14 +378,14 @@ def plot_polar_chart(
     fig.set_size_inches(size[0] / dpi, size[1] / dpi)
     if snapshot_path:
         fig.savefig(snapshot_path)
-    if show:
+    if show and not is_notebook():
```

Comment:
```suggestion
    if show:
```

Reply:
@SMoraisAnsys Thanks. I missed that one.